tmp.go

```diff
@@ -0,0 +1,9 @@
+package moira
+
+type tmp struct {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
type `tmp` is unused (unused)